Bottom: ecbdf33b90501888889a6197fafe1ed9db9455f3
Top:    9b5bcaad5b09650901f3a3c17bd97f7d1fd204c7
Author: Ian Kent <raven@themaw.net>
Date:   2013-07-30 19:15:52 +0800

Refresh of autofs-5.0.7-try-and-cleanup-after-dumpmaps.patch

---

diff --git a/CHANGELOG b/CHANGELOG
index cadf5d2..1cf79be 100644
--- a/CHANGELOG
+++ b/CHANGELOG
@@ -58,6 +58,7 @@
 - add after sssd dependency to unit file.
 - dont start readmap unless ready.
 - fix dumpmaps multi output.
+- try and cleanup after dumpmaps.
 
 25/07/2012 autofs-5.0.7
 =======================
